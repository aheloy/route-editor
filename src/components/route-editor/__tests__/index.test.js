import React from 'react';

import RouteEditor from '../';


test('render a RouteEditor', () => {
  const wrapper = shallow(
    <RouteEditor />
  );
  expect(wrapper).toMatchSnapshot();
});
