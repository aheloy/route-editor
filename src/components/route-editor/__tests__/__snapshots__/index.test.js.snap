// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a RouteEditor 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "flexFlow": "row nowrap",
      "margin": "auto",
      "marginTop": "100px",
      "maxWidth": "1200px",
    }
  }
>
  <styled.div>
    <SortableDotList
      dots={Array []}
      onAdd={[Function]}
      onDelete={[Function]}
      onSort={[Function]}
    />
  </styled.div>
  <div
    style={
      Object {
        "paddingLeft": "20px",
        "width": "960px",
      }
    }
  >
    <Map
      dots={Array []}
      onChange={[Function]}
    />
  </div>
</div>
`;
